ontology-engine/graph-engine_2.11/src/main/scala/org/sunbird/graph/nodes/DataNode.scala

```diff
@@ -58,7 +57,11 @@ object DataNode {
     def read(request: Request)(implicit oec: OntologyEngineContext, ec: ExecutionContext): Future[Node] = {
         DefinitionNode.getNode(request).map(node => {
             val schema = node.getObjectType.toLowerCase.replace("image", "")
-            request.getContext().put("schemaName", schema)
+            val objectType : String = request.getContext.get("objectType").asInstanceOf[String]
+            if (!objectType.equalsIgnoreCase(schema))
```

Comment:
Please use StringUtils for `equalsIgnoreCase`.

Reply:
Made suggested changes.